fcrepo-export/src/main/java/org/fcrepo/exporter/Exporter.java

```diff
@@ -84,13 +85,13 @@ private void export(final URI uri) {
             if (linkHeaders.contains(binaryURI)) {
                 exportBinary(uri);
             } else if (linkHeaders.contains(containerURI)) {
-                exportDescription(uri);
+                exportDescription(uri, uri);
```

Comment:
Or make the `describesUri` parameter a vararg so it can be supplied or not depending on whether it was required.